### DISCUSSION TITLE
빌드 후 제패토 패키지 용량 관련 문의

### DISCUSSION THREAD

**Discussion by xowjd913** [naverz/zepeto-studio-kor#319] [5. Q&A]
안녕하세요 ~

퍼블리시버튼을 통해 아웃풋을 뽑아내는데
플랫폼을 모바일로 체크 해 놓고 빌드 해도 아웃풋이 Android / iOS / OSX / Windows가 합쳐져서 빌드되는듯합니다.
용량을 줄이기 위해 혹시 Android / iOS 만 포함되도록 할 수 있는 방법이 있을까요 ?

**Comment by hyeonjin-jo** [comment]
안녕하세요.

현재는 플랫폼 전체를 빌드 하는 방식만 제공되고 있습니다.

의도하신 내용이 용량을 줄이기 위함이라면 이후에 플랫폼을 제한하는 빌드를 제공하더라도 각각의 플랫폼별 용량이 중요해서 큰 의미는 없어 보입니다.

전체 용량을 증가하는 방향성은 내부적으로 검토해 보겠습니다.

현재는 아래의 링크를 참고하셔서 포함되는 리소스의 리스트를 확인하시고 사용하지 않는 것이 있다면 제외하는 것을 추천드립니다.

https://github.com/naverz/zepeto-studio-kor/discussions/311

**Reply by cheolhui-lee** [reply]
조금 더 코멘트를 드리면,
플랫폼 별로 용량이 100mb만 되더라도 월드를 즐길 플레이어의 입장에서는 결코 적은 용량이 아닙니다.
물론 앱에서 캐쉬를 지원하고 있지만, 월드에 접속하기 위해 최초 무조건 한 번은 월드 파일을 다운로드를 받아야 하는 상황인데요.
모바일 특성 상 다양한 네트워크 환경을 감안한다면 당연히 용량이 적으면 적을 수록 좋은 점 참고 부탁드립니다.

**Comment by x4cf** [comment]
제가 제작한 치즈버거서버의 경우 최종빌드가 12mb정도 나와요!
리소스를 잘 활용하시면 멋진 월드를 제작하기에 충분한 용량인거같아요 :D
